tests/codegen_testcases/yul/ewasm_builtin.sol

```diff
@@ -1,62 +0,0 @@
-// RUN: --target ewasm --emit cfg -Onone --no-cse
-
-contract Testing {
-
-// BEGIN-CHECK: Testing::Testing::function::builtins__uint256
-    function builtins(uint256 arg1) public view  {
```

Comment:
If we are still maintaining EVM for the language server, I suggest that this file remain as a test for the language server, verifying the correctness of the AST for all these builtins.

In addition, some of the builtins listed here are available for other targets (like `gas()`, `address()`, `caller()`, and others). They still need to be tested for code generation, but for another target.

Reply:
So the reason I removed it is because the option `--target ewasm` was gone, and I had not added a new option `--target evm`. 

Now I've changed it so that `solang --target evm` is supported, but it is not documented in `solang compile --help` nor in the documentation,